### PR TITLE
added Badge comp

### PR DESCRIPTION
**Pull request by prateek1361**

### CODE REVIEW

File: components/alert.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+              <a href="https://github.com/">GitHub</a>
+            </button>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-container">
+      <section>
+    <ul>
```
Comment (Aditi0505) [inline components/alert.html:30]: Can fix indentation here.

File: components/alert.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          <strong>Warning!</strong> This is a Warning Alert   
+          
+      </div>
+      <div> <img class='code-img' src="../assets/alert.png"/></div>
+
+          
+      </section>
+     
+       
+
```
Comment (Aditi0505) [inline components/alert.html:87]: Can remove extra spacing here.

File: components/avatar.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Prateek sharma" />
+    <meta name="description" content="Vintage UI - A CSS component library." />
+    <title>Vintage UI</title>
```
Comment (Aditi0505) [inline components/avatar.html:9]: Suggestion: You can add the component name here to make it more meaningful. E.g. Vintage | Avatar

File: components/badge.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+         </div>
+         <div><img class='code-img' src="../assets/badge.png"/></div>
+    
+
+
+      
+            
+                
+            
+
```
Comment (Aditi0505) [inline components/badge.html:110]: Please remove extra spacing.

File: components/badge.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+            <img class="round-avatar-badge"  src="https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" alt="">
+            <span class="badge badge-offline"></span>
+        </div>
+         </div>
+         
+         <h4>Badge on Icons</h4>
+         <div class="component-area">
+            <div class="icon-w-badge">
+             
+                <img class="icon-badge" src="https://media.istockphoto.com/vectors/shopping-cart-icon-isolated-on-white-background-vector-id1206806317?s=612x612"alt="">
```
Comment (Aditi0505) [inline components/badge.html:91]: Add alt description for accessibility

File: main.css [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+  .icon-w-badge{
+    position: relative;
+   
+  }
+  .badge-on-icon{
+    border-radius: 50%;
+    position: absolute;
+    top: -3px;
+    right: -1px;
+    background-color: rgb(0, 195, 255);
```
Comment (Aditi0505) [inline main.css:148]: Can use hex code for consistency.

File: main.css [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+.Badges{
+    text-align: center;
+    width:50rem;
+   }
+   .component-area{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 13rem;
+    background-color: antiquewhite;
```
Comment (Aditi0505) [inline main.css:93]: Can use hex code for consistency.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Rest looks good to me.
Also, please add description and screenshots for better visualization.
Good work on using sematic tags.

File: main.css [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+    border-radius: 50%;
+    position: absolute;
+    top: -3px;
+    right: -1px;
+    background-color: rgb(0, 195, 255);
+  }
+
+  
+
+
```
Comment (SanketDhabarde) [inline main.css:153]: I see that you are using a common CSS file for components. I suggest making separate CSS files for every component. or you can at least put comments to understand styles for a specific component.

**Pull request merged**